Tools/wasm/emscripten/web_example/python.worker.mjs

```diff
@@ -59,29 +61,42 @@ const stderr = (charCode) => {
 
 const stdinBuffer = new StdinBuffer()
 
-var Module = {
+const emscriptenSettings = {
     noInitialRun: true,
     stdin: stdinBuffer.stdin,
     stdout: stdout,
     stderr: stderr,
     onRuntimeInitialized: () => {
         postMessage({type: 'ready', stdinBuffer: stdinBuffer.sab})
+    },
+    async preRun(Module) {
+        // TODO: remove fixed version number
```

Comment:
Unless it's going to be a major impediment to progress, I think we're going to need a solution for this before we can merge this PR - even though it's an example, it's code that will easily get forgotten along the way.

Reply:
Well the way that I wanted to do this was blocked by
https://github.com/emscripten-core/emscripten/issues/22980
and I was waiting to see if it got fixed quickly or if I should find a workaround. Emscripten 3.1.73 was released with a fix so I think we can bump the minimum suggested Emscripten version to 3.1.73 and use the straightforward approach.

Reply:
I suppose one simple fix would be to template the version into `python.sh` and then pass it in as an argument or environment variable. But I was thinking to get it from `_Py_version`.

Reply:
Okay I fixed this in 7c61ed16707e6128b352575d57b6d267538ef3bc.